types/react-scroll/modules/components/Link.d.ts

```diff
@@ -13,8 +13,8 @@ export interface ReactScrollLinkProps {
     onClick?(): void;
     duration?: number | string | ((distance: number) => number) | undefined;
     absolute?: boolean | undefined;
-    onSetActive?(to: string): void;
-    onSetInactive?(): void;
+    onSetActive?(to: string, element: HTMLElement | null): void;
```

Comment:
element shouldnt be non null per feedback.

Reply:
done, thanks for the feedback